src/components/ebay-toggle-button-group/examples/columns.marko

```diff
@@ -0,0 +1,26 @@
+<ebay-toggle-button-group columnsSM=3 columnsXS=2 columnsMD=6 columnsXL=8 ...input>
```

Comment:
The default state of this example that has columns defined (see markup) doesn't match the options underneath that are all `null`.  That might be confusing to devs.

<kbd><img width="1347" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/2e4c718a-1193-4289-b5e2-cf5c8d40f753"></kbd>

Reply:
Yes
This is fine. This is because we are setting it explicitly on the code
See show code button
We need an example with it set by default.